e2e/testsuite/testsuite.go

```diff
@@ -143,13 +131,76 @@ func (s *E2ETestSuite) GetRelayerUsers(ctx context.Context, chainOpts ...ChainOp
 // with E2ETestSuite.StartRelayer if needed.
 // This should be called at the start of every test, unless fine grained control is required.
 func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channelOpts func(*ibc.CreateChannelOptions), chainSpecOpts ...ChainOptionConfiguration) (ibc.Relayer, ibc.ChannelOutput) {
-	chainA, chainB := s.GetChains(chainSpecOpts...)
-	r := s.ConfigureRelayer(ctx, chainA, chainB, channelOpts)
```

Comment:
do we still need the `ConfigureRelayer` function if its contents have been moved here?

Reply:
we still need it for now as it's used in the wasm tests [here](https://github.com/cosmos/ibc-go/blob/31c5ff4f3e4288145354fdf59dff4e9aeef3c628/e2e/tests/wasm/grandpa_test.go#L89-L91)

This can be refactored in a follow up I think.